docs/contributing.md

```diff
@@ -0,0 +1,11 @@
+# Code Review
+
+The following checklist should be used during code reviews:
+
+- [ ] Code accomplishes its functional purpose.
+- [ ] Code passes automated testing (none at the moment)
+- [ ] Code passes linting (in progress).
+- [ ] Code is readable and maintainable.
```

Comment:
Should we add an item about code comments?

Reply:
Mmm, sure. 

There is a check below to make sure that functions are accompanied with documentation (comments), but I think comments are good as well when appropriate.

There's actually quite a craft when it comes to writing good comments: https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/

I think we can add a check like this?

  - [ ] Code comments follow generally accepted best practices.

Reply:
Done!